app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoRepo.kt

```diff
@@ -72,6 +74,11 @@ open class PocketVideoRepo(
 
     private suspend fun updateInner() {
 
+        if (!localeIsEnglish.invoke()) {
+            _feedState.postValue(FeedState.Inactive)
```

Comment:
One nit is that we hit this check all the time on the original timer, even though changing a locale is a very rare situation. Maybe we add a retryDelay here, to way less frequently for locale change? The flip side is that users changing their locale will not automatically get new content, but they'll see the "Try again" button and can force a refresh.

Reply:
This seems to be pretty cheap (in a quick test on the Cube, `localeIsEnglish.invoke` took about a quarter of a MS), so my inclination would be to keep the code as simple as possible.  That said, let me know if you feel strongly about it and I can cache the results.